src/constants.js

```diff
@@ -19,5 +19,10 @@ const ProdConstants = {
 	AIO_CLI_API_KEY: 'aio-cli-console-auth',
 	SMS_BASE_URL: 'https://graph.adobe.io/api-admin',
 };
-
+const TMOConstants = {
+	TMO_STAGE_URL: 'https://tigraph-dev.adobe.io/api',
+	TMO_SANDBOX_URL: 'https://tigraph-sandbox.adobe.io/api',
+	TMO_PROD_URL: 'https://tigraph.adobe.io/api',
+};
 module.exports = clientEnv === 'stage' ? StageConstants : ProdConstants;
+module.exports = { ...module.exports, TMOConstants };
```

Comment:
```suggestion
const envConstants = clientEnv === 'stage' ? StageConstants : ProdConstants;
module.exports = { ...envConstants, TMOConstants };
```

I wouldnt suggest reassigning `module.exports`. Even though it works, it is not a standard.

Reply:
Thanks for the suggestion @revanth0212. Will make the change and do a retest. I see people suggesting both ways, wasn't sure what the right one was. Thank you for the explanation